app/tests/reader_studies_tests/resources/ground_truth.csv

```diff
@@ -1,4 +1,4 @@
-images,foo,bool,choice,mchoice
-im1,'"yes"','true','"option1"','["option1", "option2"]'
-im2,'"no"','false','"option1"','["option1", "option2"]'
-im3;im4,'"yes"','true','"option1"','["option1", "option2"]'
+images,foo,bool,choice,mchoice,foo_explanation
```

Comment:
Does this work if there is a space in the key? Should we use 2 underscores to be consistent with django lookups?

Reply:
Spaces are fine. The double underscore is a great suggestion!